modules/sdk/node-build/index.mjs

```diff
@@ -0,0 +1,24 @@
+import getGlobalImports from './configuration/getGlobalImports.mjs';
+import getGlobalSymbols from './configuration/getGlobalSymbols.mjs';
+import getProjectExports from './configuration/getProjectExports.mjs';
+import getProjectWebContextPath from './configuration/getProjectWebContextPath.mjs';
+import bundleJavaScriptAndCSS from './esbuild/bundleJavaScriptAndCSS.mjs';
+import writeCSSLoaderJavaScriptModules from './cssLoad/writeCSSLoaderJavaScriptModules.mjs';
+
+const [
+	globalImports,
+	globalSymbols,
+	projectExports,
+	projectWebContextPath
+] = await Promise.all([
+	getGlobalImports(),
+	getGlobalSymbols(),
+	getProjectExports(),
+	getProjectWebContextPath()
+]);
+
+await Promise.all([
+	bundleJavaScriptAndCSS(globalImports, globalSymbols, projectExports),
+	writeCSSLoaderJavaScriptModules(projectExports, projectWebContextPath),
```

Comment:
Just for being overly clear, what is the difference of "bundling CSS" and writing a "CSS loader module"?

Reply:
Bundling CSS is creating a single CSS files composed of several parts (esbuild and webpack do it, for example).

A CSS loader module is a JS module we write so that when someone does `import 'my.css';` that JS module kicks in and injects HTML in the current page to load the CSS file that was bundled.

Reply:
We've been using that technique since AMD times.